src/hash.h

```diff
@@ -1,21 +1,21 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
-// Copyright (c) 2009-2017 The Bitcoin Core developers
-// Copyright (c) 2019-2021 The Gridcoin Core Developers
+// Copyright (c) 2009-2020 The Bitcoin Core developers
```

Comment:
nit, we lost Gridcoin developers here

Reply:
We don't have any original code in this file other than the http -> https change for the copyright header.

Reply:
I agree.